service/src/main/java/datawave/microservice/querymetric/handler/ShardTableQueryMetricHandler.java

```diff
@@ -605,6 +605,24 @@ public T toMetric(EventBase event) {
                         } catch (Exception e) {
                             log.error(fieldName + ":" + fieldValue + ":" + e.getMessage());
                         }
+                    } else if (fieldName.equals("EVALUATED_COUNT")) {
+                        try {
+                            long l = Long.parseLong(fieldValue);
+                            if (l > m.getEvaluatedCount()) {
```

Comment:
I am a little confused about this.  Why assume 1 evaluation if it is 0 or less?  Might make sense for a seek but not evaluation.  Same comment for the rejected count.  @billoley Can you explain this logic in the seek count ?

Reply:
Sir, that's a lower case L (for long), not a ONE 1.